tests/errors.js

```diff
@@ -17,3 +17,52 @@ test('Form sets errors from prop', () => {
 
   expect(wrapper.find(Input).props().error).toEqual('failed because blah');
 });
+
+
+test('Form sets custom error message from prop', () => {
+  const wrapper = shallow(
+    <Form rules={{test: 'required|min:8'}} errorMessages={{'min.test': 'I am a custom message!', 'min.required': 'wow, so field, much required.' }}>
+      <Input name="test" />
+      <button submit className="button"></button>
+    </Form>
+  );
+
+  wrapper.find(Input).simulate('change', { target: { name: 'test', value: 'fail' }});
+  wrapper.find(Input).simulate('blur');
+  expect(wrapper.find(Input).props().error).toEqual('I am a custom message!');
+});
+
+test('Form sets custom error message on submit', () => {
+  const wrapper = shallow(
+    <Form initialValues={{test: '' }} rules={{test: 'required'}} errorMessages={{ 'min.test': 'I am a custom message!', 'required.test': 'wow, so field, much required.' }}>
+      <Input name="test" />
+      <button submit className="button"></button>
+    </Form>
+  );
+  wrapper.find('.button').simulate('click');
+  expect(wrapper.find(Input).props().error).toEqual('wow, so field, much required.');
+});
+
+test('Form sets custom error message when an errorMessage key is not used', () => {
```

Comment:
I see, let's keep it then. I just published 1.0.0 since this is stable now. Won't do any major version bumps after this hopefully!